### DISCUSSION TITLE
Custom repo/plugin folder

### DISCUSSION THREAD

**Discussion by ealap** [marlonrichert/zsh-snap#86] [Q&A] [answered]
Default installation dumps all sourced repos in my home folder. I maybe totally missing it from the documentation and sample .zshrc, but how do I define a custom folder for repos and plugins?

**Comment by marlonrichert** [comment, marked as answer]
```zsh
% source /path/to/zsh-snap/install.zsh 

Configuring Znap...

Where do you (want to) keep your repos and/or plugins?
> ~/Git  # <-- HERE
```

**Reply by chrishoage** [reply]
@marlonrichert It would be nice if the possible configuration options were documented somewhere. 

The way I am setting up zsh-snap is perhaps a little non-standard but I have it in a submodule with my dotfiles so that i can `stow` it to the correct spot. 

By default zsh-snap appears to use the same directory that znap lives. However I wanted my repos to be in a different spot as to not clutter the git repository my dotfiles are in. 

I had to dig though the install script referenced above to find `zstyle ':znap:*' repos-dir`. 

Please consider adding all the possible configuration options (if there are more) to the readme! 

Thanks for your work on this project! I'm currently evaluating replacing zplug with znap!

**Reply by marlonrichert** [reply]
#158
